### DISCUSSION TITLE
Can't login in the qFieldSync plugin

### DISCUSSION THREAD

**Discussion by Martinetta** [opengisch/QField#3763] [Q&A]
I successfully used qGIS on a pc till yesterday, and synchronized the projects on my android phone using the FieldSync plugin with qFieldCloud. 
Now I started on OSGeoLive (lubuntu), and now I'm not able to log-in in the qFieldSync plugin.
Clicking on the blue cloud icon, the window says "Signed out", and apparently there is no way to sign-in.
It always worked on pc, can't belive it doesn't works on linux!! Surelly I'm doing something wrong.
What should I do?

**Comment by suricactus** [comment]
Hi @Martinetta . It surely does work on linux, the plugin development is on Linux. Are you sure you type your credentials correctly?

**Reply by Martinetta** [reply]
Thanks @suricactus , even if I'm pretty sure that the pw was correct, I tried to check, but I can't (I'm not able to) write the credentials again, because all the buttons in the QFieldCloud window are grey. Can't find another place where to write the credentials.
I don't remember if it asked me to write my username.
I tried to uninstall the plugin, switch off everything, switch on and reinstall the plugin, but it doesn't ask me the username again, always asks to the pw and than, clicking on the blue cloud icon,  says "signed out".

**Comment by Martinetta** [comment]
Solved in the most brutal way: uninstalling qGIS, switching the computer off, keeping it off during 1 minute, reopening and reinstalling everything, now the log-in window correctly appeared, and the  plugin works perfectly.

**Reply by suricactus** [reply]
Sometimes a bit of brutality brings peace :)

**Comment by roryfogerty** [comment]
Same on MacOS. Best to avoid this plugin on MacOS until a solution is found. An alternative is [Mergimmaps](https://merginmaps.com/) that works seamlessly

**Reply by BoswachterMarc** [reply]
I am using Ubuntu and QField(sync) since years; it runs smoothly

**Reply by roryfogerty** [reply]
Thanks, I know it runs on Linux, I dont have a Linux laptop, I have a Macbook and it doesn't work on that.

**Reply by suricactus** [reply]
@roryfogerty sometimes on MacOS the login dialog gets into the background of the main QGIS window.

**Reply by mbernasocchi** [reply]
@roryfogerty I think your `Best to avoid this plugin on MacOS` is rather strong. QFieldSync is regularly used by numerous MacOS users with success.

**Reply by mbernasocchi** [reply]
By the way, we just released a new version that should take care of this for good.
Happy to hear your feedback

**Comment by roryfogerty** [comment]
Hi, thanks for your reply. Found it and all is well. Would be good to
mention something in the documentation..

On Fri, 17 Nov 2023 at 12:43, Ivan Ivanov ***@***.***> wrote:

> @roryfogerty <https://github.com/roryfogerty> sometimes on MacOS the
> login dialog gets into the background of the main QGIS window.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/opengisch/QField/discussions/3763#discussioncomment-7593508>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABZUX4KM465LPAG42CEQT4TYE2QI3AVCNFSM6AAAAAATGCTNROVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TKOJTGUYDQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
M: 021 314933



The information contained in this E-mail is CONFIDENTIAL and may be LEGALLY
PRIVILEGED, intended only for the recipient named above.  If you are not
the intended recipient, you are notified that any use, review,
dissemination, distribution or copying of this message is strictly
prohibited.  If you have received this message and attachments in error,
please notify the sender and destroy the original message. Please note the
views that are expressed in this E-mail do not necessarily reflect those of
the company.

(Please consider the environment before printing this e-mail)

**Reply by suricactus** [reply]
Thanks for the reply! Will consider mentioning it.

**Comment by mbernasocchi** [comment]
To summarise, 
QFieldSync is mainly developed on Linux, so for sure, it works on it, possibly something went wrong @Martinetta. Good you could solve it by reinstalling.
As @suricactus mentioned, sometimes on MacOS the login dialog gets into the background of the main QGIS window  (personally had it once in a course). A simple `cmd` + `tab` brings the windows to the front.

Edit: the latest version released (4.6.1) should take care of this for good.
